build.cake

```diff
@@ -153,6 +153,30 @@ Task("Zip-Files")
     Zip(binDir, packagePath, files);
 });
 
+Task("Create-Chocolatey-Packages")
+    .IsDependentOn("Copy-Files")
+    .WithCriteria(() => isRunningOnWindows)
+    .Does(() =>
+{
+  // Create Cake package.
+  ChocolateyPack("./nuspec/Cake.Portable.nuspec", new ChocolateyPackSettings {
+      Version = semVersion,
+      Files = new [] {
+        new ChocolateyNuSpecContent {Source = string.Concat("./../", binDir.Path.FullPath, "/Cake.exe")},
```

Comment:
Maybe we can add a variable that holds the absolute path to binDir before ChocolateyPack (or make all directories absolute when specifying them)?

Then we should be able to simply use something like this:

```csharp
new ChocolateyNuSpecContent { Source = binDir + File("Cake.exe") }
```